v03_pipeline/lib/misc/nested_field.py

```diff
@@ -12,4 +12,7 @@ def parse_nested_field(t: hl.MatrixTable | hl.Table, fields: str):
             ]
         else:
             expression = expression[field]
+    # Parse float64s into float32s to save space!
+    if expression.dtype == hl.tfloat64:
```

Comment:
ah, this isn't actually new code.  It's something that was dropped during the refactor... it only impacts a couple of fields across all reference datasets though because we are much more manual in how we're selecting fields now and were mostly thorough with correctly specifying `float32`.